plugins/cilium-cni/cmd/cmd.go

```
@@ -915,7 +915,16 @@ func (cmd *Cmd) Del(args *skel.CmdArgs) error {
 
 	ns, err := netns.OpenPinned(args.Netns)
 	if err != nil {
-		return fmt.Errorf("opening netns pinned at %s: %w", args.Netns, err)
+		// We are not returning an error as this is very unlikely to be recoverable
```

Looking at other places in Cilium codebase, I believe we should only catch `IsNotExist` error here:

https://github.com/dylandreimerink/cilium/blob/37053923654a2170387dca6f4c71de6f68c39280/pkg/mtu/endpoint_updater.go#L233-L239

---

you know better than me fore sure, so if you agree let me know what do you prefer and I will update it accordingly

---

I could see arguments either way but let's say there's some other error code like "Permission Denied" came up, it's probably better to make that error more obvious to the user by returning it, rather than silently ignoring it? That would argue in favor of the `IsNotExist` check.

---

updated